buildenv/jenkins/testJobTemplate

```diff
@@ -385,7 +385,7 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							    sectionHeaderStyle(sectionHeaderHelpTextStyleCss)
 							}
 							choiceParam('TEST_FLAG', ['', 'JITAAS', 'AOT', 'FIPS', 'FIPS140_2', 'FIPS140_3_OpenJCEPlusFIPS', 'FIPS140_3_OpenJCEPlusFIPS.FIPS140-3'], "Optional. Only set for feature testing. (i.e.,  JITAAS, AOT, FIPS140_2, etc)")
-							stringParam('EXTRA_OPTIONS', "", "Use this to append options to the test command")
+							stringParam('EXTRA_OPTIONS', "", "Use this to append options to the test command e.g. to add extra JVM flags")
 							stringParam('JVM_OPTIONS', "", "Use this to replace the test original command line options")
```

Comment:
While you are here, consider improving L389, in keeping with your suggestion for EXTRA_OPTIONS 
"Replaces the test's original JVM options" to make it clear versus options that would be passed into some other application called in the test command (which for awareness is set via APPLICATION_OPTIONS).

Reply:
Done - LMK what you think of the wording. I've also done a little adjustment to CUSTOM_TARGET based on a discussion about `.sh` files in slack this week.

Also I note that the full description for CUSTOM_TARGET isn't showing up so I've moved the location of `'''` to make it consistent with the the `TARGET` parameter and this is the obvious difference. At the moment only the first line is visible for some reason:

![image](https://github.com/user-attachments/assets/4ae70d6e-f72f-4a79-b054-501c3801a4eb)

(Also noting that I'm not sure if there's an easy way to test this - maybe regen a test job somehow using this fork? But a revert will be easy if it's a problem)